packages/shared/src/lib/core/wallet/utils/buildFoundryOutputBuilderParams.ts

```diff
@@ -13,19 +13,16 @@ import { getSerialNumberFromAliasOutput } from '@core/activity/utils/outputs/get
 import { IIrc30Metadata } from '@core/token/interfaces'
 
 export async function buildFoundryOutputBuilderParams(
-    totalSupply: number,
-    circulatingSupply: number,
+    totalSupply: string,
```

Comment:
not the end of the world, but we cast these values to bigint validate them on the mint form, then store them as string and then cast them again here? Shouldn't we just store them as bigint

Reply:
The issue is that we get these values from `NumberInput` as a string and then set them reactively. I could cast them as a BigInt by introducing an extra variable if you want.